src/sources/eventstoredb/mod.rs

```diff
@@ -0,0 +1,129 @@
+use self::types::{
+    EventStoreDbMetricsHttpError, EventStoreDbMetricsReceived, EventStoreDbStatsParsingError, Stats,
+};
+use crate::{
+    config::{self, GlobalOptions, SourceConfig, SourceDescription},
+    shutdown::ShutdownSignal,
+    Event, Pipeline,
+};
+use futures::{stream, FutureExt, SinkExt, StreamExt};
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+
+pub mod types;
+
+#[derive(Deserialize, Serialize, Clone, Debug, Default)]
+struct EventStoreDbConfig {
+    #[serde(default = "default_endpoint")]
+    endpoint: String,
+    #[serde(default = "default_scrape_interval_secs")]
+    scrape_interval_secs: u64,
+    default_namespace: Option<String>,
+}
+
+pub fn default_scrape_interval_secs() -> u64 {
+    3
+}
+
+pub fn default_endpoint() -> String {
+    "https://localhost:2113/".to_string()
```

Comment:
Just to make sure, `2113` is the default port for local server? I've only found `2112` here https://developers.eventstore.com/server/v5/docs/server/networking/internal.html#ports, although that one seems to be used internally only.

Reply:
Yes it is. `2112` by default is in internal port not suited for public use.